src/main/java/rtg/RTG.java

```diff
@@ -48,7 +48,7 @@
     modid                    = ModInfo.MOD_ID,
     name                     = ModInfo.MOD_NAME,
     version                  = ModInfo.MOD_VERSION,
-    dependencies             = "required-after:forge@[" + ModInfo.MCF_MINVER + "," + ModInfo.MCF_MAXVER + ")" + ModInfo.MOD_DEPS,
+    dependencies             = "required-after:Forge@[" + ModInfo.MCF_MINVER + "," + ModInfo.MCF_MAXVER + ")" + ModInfo.MOD_DEPS,
```

Comment:
You can not do this. The Forge mod ID **has** to be all lowercase, just as with all mods, in 1.11.